coderd/database/modelqueries.go

```diff
@@ -200,6 +200,7 @@ func (q *sqlQuerier) GetAuthorizedWorkspaces(ctx context.Context, arg GetWorkspa
 	rows, err := q.db.QueryContext(ctx, query,
 		arg.Deleted,
 		arg.Status,
+		pq.Array(arg.WorkspaceIds),
```

Comment:
🤨

Reply:
We added WorkspaceIDs to the GetAuthorizedWorkspaces query to replace `GetWorkspacesWithIDs`.

Reply:
will revert the change, it is no longer used.